test/evergreen.yml

```diff
@@ -278,6 +278,27 @@ functions:
         set -o verbose
         ./t ${checkpoint_args} 2>&1
 
+  "checkpoint stress test":
+    command: shell.exec
+    params:
+      working_dir: "wiredtiger/test/format"
+      script: |
+        set -o errexit
+        set -o verbose
+
+        export WIREDTIGER_CONFIG='checkpoint_sync=0,transaction_sync=(method=none)'
```

Comment:
Not sure but can we avoid this export and run set something like: `CMD='WIREDTIGER_CONFIG='checkpoint_sync=0,transaction_sync=(method=none)' ./t -h WT_TEST.$i.$t -t r -r 2 -W 3 -n 1000000 -k 1000000 -C "cache_size=100MB"'`

It is just style thing. We have avoiding export for most of the other tasks as well.

Reply:
I tried to run the command by including WIREDTIGER_CONFIG in the CMD variable but it fails for some reason (brackets in the config value may be). Below is the failed patch build. 

https://evergreen.mongodb.com/task/wiredtiger_ubuntu1804_checkpoint_stress_test_patch_9a70a0cad47875afe7631318fc06182f3add2b47_5dfc4c19a4cf473bb06368d7_19_12_20_04_21_03

Reply:
Never mind. I guess something in the string need to be escaped but I am okay with the existing config.